cve_bin_tool/checkers/open_iscsi.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2023 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for open-iscsi
+
+https://www.cvedetails.com/product/88854/Motion-Project-Motion.html?vendor_id=23775
```

Comment:
I think maybe this is the wrong URL?

```suggestion
https://www.cvedetails.com/product/42438/Open-iscsi-Project-Open-iscsi.html?vendor_id=17429
```

Reply:
Indeed fixed